stdlib/2and3/pstats.pyi

```diff
@@ -0,0 +1,36 @@
+from profile import Profile
+from cProfile import Profile as cProfile
+from typing import Any, AnyStr, Dict, IO, List, Tuple, TypeVar, Union, overload
+
+_Selector = Union[AnyStr, float, int]
```

Comment:
I think that this only accepts `str`, not `bytes`/`unicode`. Should it be `Union[str, float, int]`?